Add more content (#3)
primer/contribute: content/how-to-contribute.md

---

> title: How to contribute

Reviewer: Doctocat now supports page descriptions that display slightly larger underneath the page heading.

![CleanShot 2021-09-02 at 08 22 29](https://user-images.githubusercontent.com/4608155/131871479-6fcb2f31-ce26-4d03-b07b-179548501157.png)

Maybe we could include descriptions for these pages.

```suggestion
title: How to contribute
description: ...
```

Author: Do you think all pages should have a description as the intro sentence/paragraph to make them consistent?

Reviewer: I think so. That description is also used in the `meta` tags. So it would be good for SEO too.

---

There are several ways you can contribute to and participate in Primer:

1. Participate in discussions
2. Propose a new UI pattern
3. Improve the documentation
4. Design or build new patterns
5. Give us feedback and report bugs 
6. Contribute to our open source repos

Please note that all contributions to Primer need to follow our [code of conduct](https://github.com/github/primer/blob/main/CODE_OF_CONDUCT.md). 

If at any time you get stuck or need help, head to #primer on Slack or start a discussion in this repo with your question.

> ## 1. Participate in discussions

Author: I think I should change these titles to remove the numbers as it looks weird in the ToC 🤔

Reviewer: Agreed 👍

Read the Primer guidelines for [writing design documentation](https://primer.style/contribute/writing-design-docs).

## 4. Design or build new patterns

You can contribute to new patterns either by design, prototype and build proofs of concept, test in dotcom, and contribute directly to our Primer open source repos.

Please read the [guidelines on designing Primer patterns](https://primer.style/contribute/design).

From the [Primer project board](https://github.com/github/primer/projects/1) (GitHub staff only), you can grab anything from **Unprioritized backlog** and **Up next** — just reach out to us first in Slack or the issue itself.

## 5. Give us feedback and report bugs 

If you spot something that doesn’t look right in our design, code or documentation, open an issue and we'll triage it accordingly.

## 6. Contribute to our open source repos

When contributing to Primer open source repos, please follow the repo's contributing guidelines:

- [Primer CSS](https://github.com/primer/css/blob/main/CONTRIBUTING.md)
- [Primer React](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md)

> - [Primer ViewComponents](#)

Reviewer: ```suggestion
- [Primer ViewComponents](https://primer.style/view-components/contributing)
```

> - [Primer ViewComponents](https://primer.style/view-components/contributing)

Reviewer: ```suggestion
- [Primer ViewComponents](https://primer.style/view-components/contributing)
- [Octicons](https://github.com/primer/octicons/blob/main/CONTRIBUTING.md)
```

---
title: How to contribute

> description: Contributions to Primer can take many different formats to align with your skills and time availabe.

Reviewer: ```suggestion
description: Contributions to Primer can take many different formats to align with your skills and available time.
```

---

There are several ways you can contribute to and participate in Primer:

1. Participate in discussions
2. Propose a new UI pattern
3. Improve the documentation
4. Design or build new patterns
5. Give us feedback and report bugs 
6. Contribute to our open source repos

All contributions to Primer need to follow our [code of conduct](https://github.com/github/primer/blob/main/CODE_OF_CONDUCT.md). 

> If at any time you get stuck or need help, head to #primer on Slack or start a discussion in this repo with your question.

Author: ```suggestion
If at any time you get stuck or need help, head to #primer on Slack or start a discussion in github/primer with your question.
```
